ntoskrnl/kdbg/kdb_symbols.c

```diff
@@ -356,7 +356,7 @@ KdbSymInit(
         CHAR YesNo;
 
         /* By default, load symbols in DBG builds, but not in REL builds */
-#if DBG
+#if DBG && defined(_M_IX86)
```

Comment:
I would prefer that an explanation in the comments (or just in the commit description) is present alongside with the code change so that people don't have to rely on guesses why symbols loading has to be done in x86 only.

Reply:
Done. TLDR: "It doesn't work"